keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,321 @@
+
+Through this proposal SIG Release's aim is to give contributors more
+flexibility to decide how to invest their time. It is explicitly *not* to push
+contributors in doing more.
+
+#### SIG Release members
+
+By applying a cadence of 3 releases per year, SIG Release members will gain a
+reduced management overhead. There are also only 3 patch releases to maintain,
+which right now can overlap up to 4. SIG Release will gain more time to ensure
+a seamless transition from the previous release team to the next one. It is
+also possible to include more shadows if the role leads conclude that this is
+appropriate.
+
+### Risks and Mitigations
+
+#### Concentrating risk
+
+In theory a reduced release cadence will cause more changes for every release.
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
+about an overall release cycle enhancement of 3-4 weeks, then we believe that
+SIG Release is able to mitigate this risk over multiple releases.
+
+#### Attention to dependencies
+
+Having fewer releases will introduce the risk of missing dependencies—for
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
+
+## Design Details
+
+### Schedule Policy
+
+The exact details of the release schedule are created by the team leads before
+the actual cycle begins. SIG Release only provides a lightweight policy, which
+is defined as:
+
+1. The first Kubernetes release of a year should start at the second or third
+   week of January to provide people more room after coming back from the
+   Christmas holidays.
+
+2. The last Kubernetes release of a year should be finished by the middle of
+   December.
+
+3. A Kubernetes release cycle has a length of of ~15 weeks.
+
+4. Events like KubeCon will be considered as blocked from development or
+   decision-making. SIG Release will also consider the week before and after
+   the event in the same way.
+
+5. An explicit break of at least two weeks between each release cycle will be
+   enforced.
+
+   This does not mean that zero development can happen during that time.
+   Rather, SIG Release will use this time to do the release retrospective and
+   plan for the next cycle.
```

Comment:
is this a sig-release break or a project break?
what is the impact on developers that sig-release takes this break?

Reply:
Wait, why?  Not that we don't usually have this break de-facto, but what's the value in enforcing one if the new RT is ready to go?

Reply:
The new RT is generally not ready to go as the RT as a whole works until the end of the cycle. I read this as giving the team explicit time to regroup and set the next team up for success by not plowing forward without a break.

Reply:
@aojea yes, this is meant to be a SIG Release brek.

@jberkus In fact we already did this, my idea was to document it more clearly to the community.

Reply:
OK.  But again ... leave it out of the KEP.  The KEP should only document the things that are changing as a consequence of 4-month releases.  If we add in a lot of extraneous stuff, it makes it impossible for folks to evaluate it.

Reply:
I rephrased the point a bit, I'm not sure about removing it. The `Schedule Policy` should be used mainly by SIG Release members to plan the cycle. It was an unwritten rule to have a break, but now we have it within the policy. :upside_down_face: